lib/internal/errors.js

```diff
@@ -114,6 +114,7 @@ E('ERR_ARG_NOT_ITERABLE', '%s must be iterable');
 E('ERR_ASSERTION', (msg) => msg);
 E('ERR_INVALID_ARG_TYPE', invalidArgType);
 E('ERR_INVALID_CALLBACK', 'callback must be a function');
+E('ERR_INVALID_CURSOR_POS', 'Can\'t set cursor row without setting its column');
```

Comment:
Would be great to avoid using contractions in the error messages.

```js
E('ERR_INVALID_CURSOR_POS',
   'Cannot set cursor row without setting its column');
```